packages/patternfly-4/react-charts/src/components/ChartVoronoiContainer/ChartVoronoiContainer.tsx

```diff
@@ -143,16 +140,18 @@ export const ChartVoronoiContainer: React.FunctionComponent<ChartVoronoiContaine
   labelComponent = allowTooltip ? <ChartTooltip /> : undefined,
   ...rest
 }: ChartVoronoiContainerProps) => {
-  const chartClassName = getClassName({className});
+  const chartClassName = getClassName({ className });
   const chartLabelComponent = React.cloneElement(labelComponent, {
     constrainToVisibleArea,
     theme,
-    ...labelComponent.props,
+    ...labelComponent.props
   });
 
   // Note: theme is required by voronoiContainerMixin, but @types/victory is missing a prop type
   // @ts-ignore
-  return <VictoryVoronoiContainer className={chartClassName} labelComponent={chartLabelComponent} theme={theme} {...rest} />;
+  return (
```

Comment:
Ahh, @redallen it looks like it was this `@ts-ignore` not being applied anymore because the offending code was moved down a line by prettier. I'll add a `prettier-ignore` on the same line, and we might have to do that in the future where we need `@ts-ignore` on a long line.

It's weird that we can't just `ts-ignore` multiple lines... looks like an open issue: https://github.com/Microsoft/TypeScript/issues/19573